docs/projects/cn0540/index.rst

```diff
@@ -355,7 +355,7 @@ HDL related
 Software related
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
Could you please add as well the two device trees? 

- https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/intel/socfpga/socfpga_cyclone5_de10_nano_cn0540.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-coraz7s-cn0540.dts